code/game/objects/items/stacks/stack.dm

```diff
@@ -201,31 +201,31 @@
 			to_chat(usr, "<span class='warning'>\The [R.title] won't fit here!</span>")
 			return FALSE
 
-		if(R.one_per_turf && (locate(R.result_type) in usr.drop_location()))
+		if(R.one_per_turf && (locate(R.result_type) in get_turf(src)))
 			to_chat(usr, "<span class='warning'>There is another [R.title] here!</span>")
 			return FALSE
 
-		if(R.on_floor && !istype(usr.drop_location(), /turf/simulated))
+		if(R.on_floor && !istype(get_turf(src), /turf/simulated))
 			to_chat(usr, "<span class='warning'>\The [R.title] must be constructed on the floor!</span>")
 			return FALSE
 
-		if(R.no_cult_structure && (locate(/obj/structure/cult) in usr.drop_location()))
+		if(R.no_cult_structure && (locate(/obj/structure/cult) in get_turf(src)))
 			to_chat(usr, "<span class='warning'>There is a structure here!</span>")
 			return FALSE
 
 		if(R.time)
 			to_chat(usr, "<span class='notice'>Building [R.title] ...</span>")
```

Comment:
While you're here could you remove the space after `[R.title]`? I never actually noticed it until just now but it'll bug me now that I have.
![image](https://user-images.githubusercontent.com/57483089/125189892-4a6f9180-e232-11eb-93af-a5fa24155cdb.png)